ga4gh/backend.py

```diff
@@ -570,6 +580,15 @@ def callSetsGenerator(self, request):
             request, variantSet.getCallSetIdMap(),
             variantSet.getCallSetIds())
 
+    def genotypePhenotypeGenerator(self, request):
+        """
+        Returns a generator over the (genotypePhenotype, nextPageToken)
+        pairs defined by the specified request.
+        """
+        intervalIterator = GeneotypePhenotypeIterator(
```

Comment:
@jeromekelleher Hi.   Thanks again for the thoughtful feedback.  On this item, I've looked into removing GeneotypePhenotypeIterator.  As you mentioned, it is heavyweight for the actual work it needs to do.  As far as I can tell, if I remove this class, it will mean implementing an equivalent of `runSearchRequest` in `searchGenotypePhenotype`.  I have no problem doing this as the g2p request is quite different than the reads/referenceSets/etc.  I just wanted to see if there were any objections to this approach.